templates/base.html.twig

```diff
@@ -24,6 +24,31 @@
 
     {% block stylesheets %}
         {{ encore_entry_link_tags('app') }}
+            <style>
+                {#
+                This is needed so the magazine names in the subscription panel are correctly overflowing.
+                The max width depends on the max-width of the whole container, which is controlled by a setting (KBIN_PAGE_WIDTH)
+                 #}
+                .subscription-list .subscription a {
+                    {% if app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_SUBSCRIPTIONS_IN_SEPARATE_SIDEBAR')) is same as 'true' %}
+                        {% if app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_PAGE_WIDTH')) is same as constant('App\\Controller\\User\\ThemeSettingsController::MAX') %}
+                            max-width: calc(100vw / 9 - 3rem);
+                        {% elseif app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_PAGE_WIDTH')) is same as constant('App\\Controller\\User\\ThemeSettingsController::AUTO') %}
+                            max-width: calc(85vw / 9 - 3rem);
+                        {% else %}
+                            max-width: calc(1360px / 9 - 3rem);
+                        {% endif%}
+                    {% else %}
+                        {% if app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_PAGE_WIDTH')) is same as constant('App\\Controller\\User\\ThemeSettingsController::MAX') %}
+                            max-width: calc(100vw / 4 - 3.5rem);
+                        {% elseif app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::KBIN_PAGE_WIDTH')) is same as constant('App\\Controller\\User\\ThemeSettingsController::AUTO') %}
+                            max-width: calc(85vw / 4 - 3.5rem);
+                        {% else %}
+                            max-width: calc(1360px / 4 - 3.5rem);
+                        {% endif%}
+                    {% endif %}
+                }
+            </style>
```

Comment:
highly recommended to find a way to put this rule into the stylesheet

from what I could see, setting page width to either auto or max would set `.width--{auto,max}` on the parent container of the subscription sidebar, and when subscription sidebar is inline it'll have `.inline` class set, so I think it should be possible to define the default values for fixed width container and then add rules after to make setting specific override:

```scss
.subscription-list {
  /* defaults for fixed width container */
}

// setting specific override rules
.subscription-list {
  .width--auto & {
    // auto container panel override
    &.inline {
      // auto inline panel override
    }
  }
  .width--max & {
    // max container panel override
    &.inline {
      // max inline panel override
    }
  }
}
```

Reply:
I'll look into that. That would be a far better solution, I agree